src/commands/utils/color.ts

```diff
@@ -12,6 +12,14 @@ function getHexCodeForColor(colorName: string) {
 	return null;
 }
 
+function getNamesForHexCode(hexCode: string) {
+	hexCode = hexCode.toUpperCase();
+	if (hexCode in namedColors) {
+		return (namedColors as { [key: string]: string[] })[hexCode];
```

Comment:
i hate this please help where should i type it properly

Reply:
Turn the JSON file into a module